packages/react-native-editor/.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Those GitHub template files might not work anymore. Should they be all deleted or moved to the top of the repository?

Reply:
Deleted all the issue and PR, they are already present in gutenberg-mobile and can be used there.